airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2357,6 +2357,13 @@ components:
           type: integer
           readOnly: true
           description: The number of free slots at the moment.
+        description:
+          type: string
+          description: |
+            The description of the pool.
+
+            *New in version 2.3.0*
```

Comment:
(I'm still continually sad that there isn't a way in OpenAPI spec to specify this more than just free-text.)

Reply:
OpenAPI seems not to be interested in API versioning and the document renderers don't check for new fields/changes. It would have been better if they do because this approach is error-prone.